snapcraft/main.py

```diff
@@ -15,131 +15,82 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import argparse
+"""
+snapcraft
+
+Usage:
+  snapcraft [--version | --help] [options] COMMAND [ARGS ...]
+
+Options:
+  -h --help        show this help message and exit
+  -v --version     show program version and exit
+  -V --verbose     print additional information about command execution
+
+The available commands are:
+  list-parts   List available parts which are like “source packages” for snaps.
+  list-plugins List the available plugins that handle different types of part.
+  init         Initialize a snapcraft project.
+  add-part     Add a part to your snapcraft.yaml, interactively presenting
+               options.
+  help         Obtain help for a certain plugin or topic
+
+The available lifecycle commands are:
+  clean        Remove content - download, build or install artifacts.
+  pull         Download or retrieve artifacts defined for a part.
+  build        Build artifacts defined for a part.
+  stage        Stage the part's built artifacts into the common staging area.
+  strip        Final copy and preparation for the snap.
+  snap         Create a snap.
+
+See 'snapcraft COMMAND --help' for more information on a specific command.
+
+For more help, visit the documentation:
+http://developer.ubuntu.com/snappy/snapcraft
+"""
+
 import pkg_resources
 import sys
 
-import snapcraft.cmds
+# Non-Standard Library modules
```

Comment:
I opted for 3 sections, is that wrong?

part of python

external packages/modules

part of snapcraft.